great_expectations/checkpoint/types/checkpoint_result.py

```diff
@@ -129,7 +127,16 @@ def list_data_asset_names(self) -> List[str]:
             )
         return self._data_asset_names
 
+    @public_api
     def list_expectation_suite_names(self) -> List[str]:
+        """Return the list of expecation suite names for a checkpoint.
+
+        Args:
+            None
+
+        Returns:
+            self._expectation_suite_names: List[str] of all expectation suite names
```

Comment:
```suggestion
            The list of expectation suite names.
```

Reply:
Fixed but my branch got messed up, closing this and opening a fresh PR.